prow/plugins.yaml

```diff
@@ -606,6 +606,15 @@ plugins:
   kubernetes-sigs/gcp-filestore-csi-driver:
   - trigger
 
+  kubernetes-sigs/cri-o:
+  - assign
```

Comment:
I think you only need `trigger`. The rest you should already get for free.

Reply:
oh nice

Reply:
fixed